token/program-2022-test/tests/confidential_transfer.rs

```diff
@@ -414,6 +377,7 @@ async fn ct_initialize_and_update_mint() {
     assert_eq!(extension.authority, None.try_into().unwrap());
 }
 
+#[cfg(feature = "proof-program")]
 #[tokio::test]
 async fn ct_configure_token_account() {
     let ConfidentialTransferMintWithKeypairs {
```

Comment:
It's totally fine, but just to be sure, you mean to remove the usage of this type as you go back through and re-enable the tests?

Reply:
Yep, all the tests behind `proof-program` needs update anyway, so I did not remove `ConfidentialTransferMintWithKeypairs` for those. Subsequent PRs will remove the type.